src/olympia/git/migrations/0006_auto_20220412_1144.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.12 on 2022-04-12 11:44
+
+from django.db import migrations
+
+
+def delete_waffle_switch(apps, schema_editor):
+    Switch = apps.get_model('waffle', 'Switch')
+    Switch.objects.filter(name='git-storage-broader-structure').delete()
```

Comment:
nit, but in the interests of DRY should we have a generic utility function that we could reuse for each time we want to delete a waffle switch?